src/nft/ZoraCreator1155Impl.sol

```diff
@@ -385,8 +376,15 @@ contract ZoraCreator1155Impl is
         uint256 quantity,
         bytes memory data
     ) external nonReentrant onlyAdminOrRole(tokenId, PERMISSION_BIT_MINTER) {
-        // Call internal admin mint
-        _adminMint(recipient, tokenId, quantity, data);
+        // If this is the token's first mint:
+        if (firstMinters[tokenId] == address(0)) {
+            // Store the recipient address as the first minter
+            // Note: If the recipient is address(0) the tx will revert in the `_mint` call below
+            firstMinters[tokenId] = recipient;
+        }
+
+        // Mint the specified tokens
+        _mint(recipient, tokenId, quantity, data);
```

Comment:
why call _mint instead of _adminMint?  Can we get rid of the `_adminMint` function?

Reply:
yep i removed `_adminMint` bc it was only used by this function and was just wrapping `_mint`